packages/react-router-dom/modules/Router.js

```diff
@@ -1 +1 @@
-export { Router as default } from 'react-router'
+export { default } from 'react-router/Router'
```

Comment:
Removing  re-exports would be a major change so maybe it is a bit drastic.

Also what tree shaking benefits would you hope to get that direct imports to not give you?

Since react-router mainly exports a single module from each file, tree shaking actually does not provide any value beyond removing unused modules/file which we achieve here through direct imports.

Linking to the CommonJS build by default is just the safer option but actually I think I can improve this.